internal/cache/controller_obj_cache.go

```diff
@@ -2650,6 +2650,14 @@ func checkPublicCloud(client *clients.AviClient) bool {
 		networkName := lib.GetNetworkName()
 		if networkName == "" && lib.GetCloudType() != lib.CLOUD_GCP {
 			// networkName is required param for AWS and Azure Clouds
+			if lib.GetCloudType() == lib.CLOUD_AWS {
+				vipNetworkList, err := lib.GetVipNetworkList()
+				if err != nil {
+					utils.AviLog.Errorf(err.Error())
```

Comment:
We can be bit more verbose here .. e.g. error in getting vip network list in AWS cloud .. then the actual error.

Reply:
added more detailed logging

Reply:
can both of these logs get merged into one?
Also - the log at 2657 - seems to say - that you are checking something - but there's no way to verify whether it was successful or not , so maybe add another log in 2663 and say that the verification was successful?